user/pages/03.commerce2/02.developer-guide/05.payments/04.create-payment-gateway/01.on-site-gateways/docs.md

```diff
@@ -1,268 +0,0 @@
----
-title: On-site payment gateways
-taxonomy:
-    category: docs
----
-
-On-site payment gateways allow the customer to enter credit card details directly on the site. The details might be safely tokenized before they reach the server (Braintree, Stripe, etc) or they might be transmitted directly through the server (PayPal Payments Pro).
-
-On-site payment flow:
```

Comment:
Where is the payment flow explanation in the new docs? It seems that we start immediately with creating payments, which is confusing, cause at that point the reader has no clue what a payment method is.